src/c20_client/get_client_id.py

```diff
@@ -0,0 +1,43 @@
+"""
+File used to check for an environment variable for id and to
+get an id form server and assign the id to the enviornment
+variable if the environment variable does not yet exist
+"""
+from os import getenv
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
I think this would be better as a class that handles both the API key and the client id - after all, both come from the `.env` file.  This would allow us to encapsulate the code, which would put this call to `load_dotenv` inside a function rather than in the global space.